app/init.php

```diff
@@ -1,19 +1,31 @@
 <?php
+// Composer autoload
 require 'vendor/autoload.php';
+
+// Initialize Dotenv library
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+// initialize Twig template engine
 $loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
 $twigConfig = [];
-if (getenv('PRODUCTION')) {
+
+// If we're in production env, use cache
+if ($_ENV['PRODUCTION'] === true) {
```

Comment:
In session, I kept telling y'all to use `getnev`, but that doesn't work here (it's been _depricated_, or removed).

You were all using the correct form : `$_ENV` to access our `.env` settings!